app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,105 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>A modern languages teacher can  be a rewarding way to use your language knowledge, while opening young minds to the languages and cultures around the world.</p>
+  <p>Teacher training is available in 18 modern languages with the most popular being French, German and Spanish in key stage 2 and 3.</p>
```

Comment:
@caitlin-reidy did we get clarification on whether this is 18 modern languages teacher training courses or 18 modern languages you can teach in schools?